src/seasidevcardmodel.cpp

```diff
@@ -257,12 +257,13 @@ void SeasideVCardModel::readContacts()
     if (vcf.open(QIODevice::ReadOnly)) {
         // TODO: thread
         QVersitReader reader(&vcf);
-
+#if QT_VERSION < QT_VERSION_CHECK(6, 0, 0)
         if (!mDefaultCodec.isEmpty()) {
             QTextCodec *codec = QTextCodec::codecForName(mDefaultCodec.toLatin1());
```

Comment:
```suggestion
            QByteArray readerCodec = mDefaultCodec.toLatin1();
            QTextCodec *codec = QTextCodec::codecForName(readerCodec.constData());
```

This way this surrounding QT_VERSION check could be removed.

Qt5.6: https://doc.qt.io/archives/qt-5.6/qtextcodec.html#codecForName-1
Qt6: https://doc.qt.io/qt-6/qtextcodec.html#codecForName-1

Reply:
QTextCodec in core5compat and not good to use it